js/downloads.js

```diff
@@ -23,7 +23,7 @@ var packagesV8 = [hadoop2p7, hadoop2p6, hadoopFree, sources];
 var packagesV9 = [hadoop2p7, hadoop2p6, hadoopFree, scala2p12_hadoopFree, sources];
 
 addRelease("2.4.4", new Date("08/30/2019"), packagesV9, true);
-addRelease("2.3.3", new Date("02/15/2019"), packagesV8, true);
+addRelease("2.3.4", new Date("09/09/2019"), packagesV8, true);
```

Comment:
Thank you, @kiszk . This should be VOTE pass day which is different from news announce day.

Reply:
Thank you. I will update the date soon.

Reply:
@dongjoon-hyun Thank you. Two `downloads.js` have been updated.